modules/ROOT/pages/dataweave-cookbook.adoc

```diff
@@ -1,4 +1,4 @@
-= DataWeave Cookbook
```

Comment:
In the future we could change the URL from dataweave-cookbook to dataweave-examples and make the redirects. Not something to do right now, but could be considered.